packages/platform-sdk/src/generated/models/cart.ts

```diff
@@ -981,11 +1007,11 @@ export type DiscountCodeState =
   | string
 export interface DiscountedLineItemPortion {
   /**
-   *	Cart Discount applicable on the Line Item.
+   *	A [CartDiscountReference](ctp:api:type:CartDiscountReference) or [DirectDiscountReference](ctp:api:type:DirectDiscountReference) for the applicable discount on the Line Item.
    *
    *
    */
-  readonly discount: CartDiscountReference
+  readonly discount: Reference
```

Comment:
@ajimae Our team got around to updating the `@commercetools/platform-sdk` and noticed that this is causing a type error. Shouldn't the type for this field still be `CartDiscountReference` instead of a generic `Reference`? I can create an issue for this change if that would be helpful.

Reply:
Yes please, you can create a new issue so we can look into it.

Thanks

Reply:
> @ajimae Our team got around to updating the `@commercetools/platform-sdk` and noticed that this is causing a type error. Shouldn't the type for this field still be `CartDiscountReference` instead of a generic `Reference`? I can create an issue for this change if that would be helpful.

This change was necessary due to the introduction of `DirectDiscounts`. Now it can be a `CartDiscountReference` or a `DirectDiscountReference`
The `Reference` type correctly covers both cases as can be seen [here](https://github.com/commercetools/commercetools-sdk-typescript/blob/570afe66dbdeeffb135258911df7c43d3c083357/packages/platform-sdk/src/generated/models/common.ts#L1025)

Reply:
Thanks! Sounds like these types are functioning as expected, so I don't think an issue is necessary. We will just make the necessary updates to our code